setup.py

```diff
@@ -26,7 +26,10 @@ def get_long_description():
     packages=find_packages(),
     dependency_links=[],
     install_requires=[
-        "django"
+        "django==1.9.6",
```

Comment:
Never do explicit version links in a setup.py file. Just list the items. Pip and the Readme handles these things better. The only version ever acceptable in a setup.py file is a range, for example if requests 1.x was crap then you list `requests>=2.0`.